test/Client.test.js

```diff
@@ -100,7 +101,7 @@ describe('Client complex inline script path', () => {
           )
           .then((requestObj) => {
             expect(requestObj.url()).toMatch(
```

Comment:
Don't modify existing tests, please add new

Reply:
ok, wait a minutes
thanks for your enthusiasm

Reply:
already add a new test
maybe it is complicated due to lack of test practice

Reply:
Don't worry code looks good so wait CI green and CLA